microcosm_fastapi/conventions/schemas.py

```diff
@@ -1,11 +1,46 @@
 from typing import Any, List, Optional
+from enum import Enum
 
 from pydantic import BaseModel as BaseModel, AnyHttpUrl, Field
 
 from microcosm_fastapi.naming import to_camel
 
 
-class BaseSchema(BaseModel):
+class EnhancedBaseModel(BaseModel):
+
+    @classmethod
+    def _get_value(
+            cls,
+            v: Any,
+            to_dict: bool,
+            by_alias: bool,
+            include,
+            exclude,
+            exclude_unset: bool,
+            exclude_defaults: bool,
+            exclude_none: bool,
+    ) -> Any:
+        # The reason that we're overriding the Pydantic's BaseModel is so that
+        # we can create our own Config parameters such as 'not_use_enum_names'
+
+        # 'not_use_enum_names' is bespoke pydantic config parameter used to indicate when
+        # enum names shouldn't be used. Because it is more common to want to use enum names
+        # the default execution flow for Enums will be to use their names.
+        if isinstance(v, Enum) and not getattr(cls.Config, 'not_use_enum_names', False):
```

Comment:
nit: use double quotes instead of single

Reply:
Done 👍